CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Changelog
```

Comment:
contrib-updater will help you with this file because it auto-fills the changelog from existing releases

Reply:
Changelog has been filled out by `contrib-updater`.